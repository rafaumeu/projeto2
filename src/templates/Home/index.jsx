import { Wrapper } from './styles'

function Home() {
  return (
    <Wrapper background='blue'>
      <h1>Hello</h1>
    </Wrapper>
  )
}

export default Home
