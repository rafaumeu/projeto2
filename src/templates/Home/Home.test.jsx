import { render, screen } from '@testing-library/react'

import Home from '.'

test('renders learn react link', () => {
  render(<Home />)
  const linkElement = screen.getByText(/hello/i)
  expect(linkElement).toBeInTheDocument()
})
