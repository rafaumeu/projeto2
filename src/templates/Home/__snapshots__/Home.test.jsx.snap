// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
.c0 {
  background: #dc143c;
}

<div
  class="c0"
>
  <h1>
    Hello
  </h1>
</div>
`;
